beefy-gadget/src/worker.rs

```diff
@@ -244,12 +321,21 @@ where
 			futures::select! {
 				notification = self.finality_notifications.next().fuse() => {
 					if let Some(notification) = notification {
+						if self.state == State::New {
+							match self.init_validator_set() {
+								Ok(()) => (),
+								Err(err) => {
+									error!(target: "beefy", "🥩 Init validator set failed: {:?}", err);
+									return;
```

Comment:
:thinking: this shouldn't be an error and it should not really stop the entire worker, should it?

Reply:
I'm not sure. It should not cause any harm to turn the error into a warning. The worker state will remain `State::New` and no voting will be done.

Reply:
Yeah, I think this should at most be a `warn`. I say "at most", because this is not really actionable by the node operator, they can only ignore this and wait for the runtime to include the pallet and in case of the transition period this will definitely be triggered.

Also if we expect the worker to remain in `State::new` we shouldn't be doing `return` here, should we?

Reply:
Will turn it into an info and continue.